### DISCUSSION TITLE
Add ability to get info on connected devices

### DISCUSSION THREAD

**Discussion by jobeylev** [danielpaulus/go-ios#218] [Ideas]
Hi, my automation engineers are using a remote setup where they can't physically see the devices they've got connected. They want to be able to programmatically determine information on each connected device, especially whether it's an iPhone or an iPad. Is it possible to add this feature?

**Comment by nanoscopic** [comment]
The `info` CLI option already provides the device type information in the form of the "ProductType". It can be decoded to determine the example iPhone/iPad model.

**Reply by jobeylev** [reply]
Thanks!
